napari_arboretum/_tests/test_plugin.py

```diff
@@ -40,7 +41,7 @@ def test_colormap_change(viewer_plugin):
     """
     viewer, plugin = viewer_plugin
     id = 140
```

Comment:
Could this `id` not be set as a variable at the top of the file, seeing as it is being used more than once?